dawn/js/components/App.js

```diff
@@ -24,6 +24,7 @@ export default React.createClass({
     this.addSteps(joyrideSteps);
     RobotInfoStore.on('change', this.updateRobotInfo);
     AlertStore.on('change', this.updateAlert);
+    this.startTour();
```

Comment:
Oops, I should have made this clearer in the issue. #211 means that we should automatically play the interactive tutorial the very first time Dawn is opened on a computer, but after the first time we should stop playing the tutorial because that will get annoying, and they will already have seen it.